backend/README.md

```diff
@@ -69,6 +69,10 @@ Install packaging tools:
 
     pip install -r requirements.txt
 
+(Optional) Install all preview builders to be able to generate preview for most previews:
```

Comment:
please indicate a non exaustive list of which kind of preview we can have with this option.

Reply:
```suggestion
(Optional) Install all preview builders to be able to generate preview for most preview: video, 3d, etc... :
```